autoload/neomake/makers/clippy.vim

```diff
@@ -0,0 +1,45 @@
+" vim: ts=4 sw=4 et
+
+" Yet to be determined
+let s:rustup_has_nightly = -1
+
+function! neomake#makers#clippy#clippy() abort
+    let errorfmt = '%Eerror[E%n]: %m,'.
+                 \ '%Eerror: %m,'.
+                 \ '%Wwarning: %m,'.
+                 \ '%Inote: %m,'.
+                 \ '%-Z\ %#-->\ %f:%l:%c,'.
+                 \ '%I\ %#\= %t%*[^:]: %m,'.
+                 \ '%I\ %#|\ %#%\\^%\\+ %m,'.
+                 \ '%-G%s,'
+
+    " When rustup and a nightly toolchain is installed, that is used.
+    " Otherwise, the default cargo exectuable is used. If this is not part
+    " of a nightly rust, this will fail.
+    if !exists('s:rustup_has_nightly') || s:rustup_has_nightly == -1
+        if !executable('rustup')
+            let s:rustup_has_nightly = 0
+            call system('rustc --version | grep -q "\-nightly"')
+            if v:shell_error
+                call neomake#utils#LoudMessage('Clippy requires a nightly rust installation.')
```

Comment:
I agree, since neomake can't function as its supposed to that should be an error. (Although I think the levels are confusingly namend when a "Quiet" message can be "heard" and a "Loud" one can't.)